cluster/juju/layers/kubernetes-worker/debug-scripts/kubectl

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -ux
+
+sudo -iu ubuntu kubectl cluster-info > $DEBUG_SCRIPT_DIR/cluster-info
```

Comment:
Is there any reason these are being run as the ubuntu user? I'm not opposed to this, just curious why